@here/harp-mapview/test/PickingRaycasterTest.ts

```diff
@@ -1,13 +1,13 @@
 /*
- * Copyright (C) 2020-2021 HERE Europe B.V.
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```

Comment:
The years where correct before right? It should go from the year the file was created to the current year. That's what the script recently written by Thomas and German does anyway.

Reply:
sorry, not sure what caused that, I did not cause that, I guess I run yarn pre-test .. I ll revert it

Reply:
done